.nuget/packages.config

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="ILMerge" version="3.0.21" targetFramework="net452" />
+  <package id="Lucene.Net" version="3.0.3" targetFramework="net40" />
+  <package id="MicroBuild.Core" version="0.2.0" targetFramework="net45" />
   <package id="Microsoft.Build" version="15.1.262-preview5" targetFramework="net45" />
+  <package id="Microsoft.DotNet.Build.Tasks.Feed" version="2.2.0-beta.19178.1" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
+  <package id="Microsoft.DotNet.Maestro.Tasks" version="1.1.0-beta.19178.1" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
   <package id="Microsoft.VisualStudio.Composition" version="15.8.98" targetFramework="net45" />
   <package id="Microsoft.VisualStudio.ProjectSystem" version="16.0.201-pre-g7d366164d0" targetFramework="net45" />
   <package id="Microsoft.VSSDK.BuildTools" version="15.9.3032" targetFramework="net46" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
-  <package id="Lucene.Net" version="3.0.3" targetFramework="net40" />
   <package id="Microsoft.Web.Xdt" version="2.1.2" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
+  <package id="NuGet.Build.Tasks.Pack" version="4.9.2" />
+  <package id="NuGet.Client.EndToEnd.TestData" version="1.0.0" />
```

Comment:
This entry is new.  All other changes in this file are just reordering entries by package ID.